pkg/agent/protocol/grpc/handler.go

```diff
@@ -61,18 +66,23 @@ func (h *handler) streamHandler(_ interface{}, serverStream grpc.ServerStream) e
 	excluded := contains(h.disruption.Excluded, serviceName)
 	if !excluded {
```

Comment:
I'm not sure if we should `nolint: nestif` this out as a `TODO`, or if this is worth refactoring it in this PR.

Reply:
Looks like we could just call `transparentForward` if excluded is true, and save a nested if. We then duplicate this call also at the en but is a single statement.

Reply:
Done, I think it does look better this way.